integration-tests/gatsby-source-wordpress/test-fns/data-resolution.js

```diff
@@ -308,4 +315,69 @@ describe(`data resolution`, () => {
     expect(gatsbyResult.data.wpPage).toStrictEqual(wpGraphQLPageNormalizedPaths)
     expect(gatsbyResult.data.wp.seo).toStrictEqual(WPGraphQLData.seo)
   })
+
+  it(`Does not download files whose size exceed the maxFileSizeBytes option`, async () => {
+    const wpPluginOpts = getPluginConfig()
+    const { maxFileSizeBytes } = wpPluginOpts.options.type.MediaItem.localFile
+    /**
+     * Ensure that the fileSize "gt" filter value matches the maxFileSizeBytes value in gatsby-config
+     */
+    const { data: { allWpMediaItem: { nodes }}} = await fetchGraphql({
+      url,
+      query: /* GraphQL */`
+        query tooLargeFiles($maxFileSizeBytes: Int!, $includedMimeTypes: [String]!) {
```

Comment:
This is awesome! What do you think about also doing the opposite by querying using `lt` and asserting that all nodes below the maxFileSizeBytes option have a localFile node

Reply:
Noice! I just added a test for this as well as asserting that files who have included mime types also have a local file node.